frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -585,3 +585,7 @@ export type SelfSubjectAccessReviewKind = {
     evaluationError?: string;
   };
 };
+
+export type ResourceEventStreamProps = {
+  obj?: K8sResourceCommon;
```

Comment:
I think `resource` is generally used for such a prop. `obj` is too vague.

Reply:
Furthermore is `obj` really optional?

Reply:
Adding a wrapper and exposing it as Resource.